src/utils/parse.rs

```diff
@@ -51,6 +51,7 @@ pub(crate) fn parse_matrix_string(string: &str) -> Result<Vec<Vec<String>>, Math
     }
 
     // delete `[[` in front and `]]` in the end and split the matrix into rows
+    let string = string.replace("], [", "],[");
```

Comment:
\s* in the RegEx  means that arbitrarily many spaces may occur, but this still restricts us to one space and may lead to errors as matrices go through the matching that are not valid.

Reply:
replaced it with \s?